packages/thumbprint-react/components/Chip/test.tsx

```diff
@@ -80,3 +83,15 @@ test('renders the correct text when clicked', () => {
     expect(button).toBeTruthy();
     expect(button).toHaveTextContent('Remove Beverages');
 });
+
+test('renders the `icon` passed in', () => {
+    const wrapper = mount(
+        <FilterChip
+            text="Lightning"
+            icon={<ContentModifierLightningTiny />}
```

Comment:
I recommend just doing `<svg />` here. It keeps the snapshot small and simplifies the test a little.

Reply:
Done